### DISCUSSION TITLE
Welcome to pvlib-python Discussions!

### DISCUSSION THREAD

**Discussion by wholmgren** [pvlib/pvlib-python#1276] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. Some things discussions might be used for:
  * Figuring out how to best use discussions
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  For the maintainers, here are some tips 💡 for getting started with Discussions. 

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by spaceguy152** [comment]
Hi @wholmgren , Recently I was working with the GFS forecast function and was able to extract the data easily. One thing I noticed is that the data is not adjusted for the timezone. Is the raw data in UTC? Do we have to again convert the raw data to the required timezone?

**Reply by wholmgren** [reply]
@spaceguy152 please see https://github.com/pvlib/pvlib-python/issues/1237 and also search the [pvlib python google group](https://groups.google.com/g/pvlib-python) for related discussion. Please open a [new discussion](https://github.com/pvlib/pvlib-python/discussions) if you have further questions.

**Reply by spaceguy152** [reply]
Thank you.

**Comment by Kaesekopf** [comment]
I am just starting with python pvlib and the basic_chain function seemed a good way to get acquainted . However, one thing really wonders me. The PV-array is only described with the module_parameters argument, and there is no parameter or keyword to describe the size of the array. 

Also the code suggests that only the dc- power of a single  module is calculated, and when I run the function for a 250 W module and a 5kW inverter the AC output is never comes above 200W, again confirming by suspicion  that only the DC output of a single module is calculated.

As a quick workaround I will make a local version of the basic_chain function with an array-inverter power ratio as additional output and a scaling of the DC output within the function. This will solve my problem for the moment, but I still have some questions:

* Am I overlooking something?
* Is this an intended feature, for example, the function is only intended to be used for so-called AC-modules (a single module connected to it's own micro-inverter). If so shouldn't this be in the documentation?
* If my suspicion is right, should this be an official issue?

**Reply by wholmgren** [reply]
@Kaesekopf please open a [new discussion](https://github.com/pvlib/pvlib-python/discussions)
